File: PHP/Daily004_easy/Password.php
<?php
/** 
 * PASSWORD GENERATOR
 *
 * challenge #4 (easy)
 * http://www.reddit.com/r/dailyprogrammer/comments/pm6oj/2122012_challenge_4_easy/
 *
 * sarthak7u@gmail.com
 */

//stores alphabets
$alphabet = range('a', 'z');

//randomly selects length for password
$passwordLength = rand(8,12);

$password = "";

for ($i = 0; $i < $passwordLength; $i++) 
{ 
	$temp = $alphabet[rand(0,25)];
	$password = $password.$temp;
}

echo $password;

?>

File: C#/Daily004_easy/Password.cs
/**
 * PASSWORD GENERATOR
 *
 * challenge #4 (easy)
 * http://www.reddit.com/r/dailyprogrammer/comments/pm6oj/2122012_challenge_4_easy/
 *
 * sarthak7u@gmail.com
 */

using System;
public class Password
{
    static void Main(string[] args)
    {
        Random gen = new Random();

        string alphabet = "abcdefghijklmnopqrstuvwxyz";
        int passwordLength = gen.Next(8,12);
        string password = "";

        for (int i = 0; i  < passwordLength; i++)
        {
            password += alphabet[gen.Next(0,25)];
        }
        Console.WriteLine(password);
    }
}
